index.html

```diff
@@ -578,6 +582,10 @@ <h3>
       <p class="ednote">
         The support for URI variables comes from the need exposed by [[WOT-TD]] to be able to describe existing <a>TD</a>s that use them, but it should be possible to write a <a>Thing Description</a> that would use <a>Action</a>s for representing the interactions that need URI variables and represent the URI variables as parameters to the <a>Action</a> and in that case that could be encapsulated by the implementations and the |options| parameter could be dismissed from the methods exposed by this API.
       </p>
+      <p>
+        The <dfn>data</dfn> property if defined, represents additional opaque
+        data that needs to be passed to the interaction.
```

Comment:
Is this related to the new `interactionData` or it is just an extension point to let implementations add other configuration parameters?

Reply:
No, it is not related to `InteractionData`, but to `InteractionOptions` only.
ATM the Event affordances may use `data` in order to pass subscription related information.
In my own opinion I'd get rid of `data` and also of `uriVariables`, but there is some kind of need for them in the TD spec.

Reply:
We could rename it to `input` in order to avoid associations to `data` used elsewhere.

Reply:
I am also unsure what `data` is about? Can you elaborate what you mean and *where* the need in TD is?
[data](https://w3c.github.io/wot-thing-description/#eventaffordance) in TD EventAffordance is what gets pushed to the consumer (e.g., alert information)

Reply:
This carries the `subscription` and `cancellation` data, and I know it name-clashes with `data`,  which is why I suggested changing the name. However, in JS `data` is pretty much a common idiom for representing opaque data in objects. However, I see this is missing from the algorithm for `subscribeEvent()`. It is covered in the "handle subscribe requests" and unsubscribe as well in ExposedThing, albeit implicitly (as they get the full `options`dictionary.

Reply:
If I rename it to `input`, the name will mess with Actions (where we don't use this :D).
But can't say `subscriptionData` either (I have considered that), since it is a generic `InteractionOptions`. So I thought that in general `InteractionOptions` could contain some opaque data. You can also think of it as future-proofing. 
In turn we need to make it clear in the algorithms how do we use that data.